app/src/main/java/com/google/droidjump/StartFragment.java

```diff
@@ -58,6 +54,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         return rootView;
     }
 
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        getView().findViewById(R.id.menu_button).setOnClickListener(ignored -> activity.openUserMenu());
```

Comment:
nit: can we use a differenet name of ignored? maybe view?

Reply:
@saimonsaret said to use this keyword if we do not use the variable.